seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/provider/SpringBootConfigurationProvider.java

```diff
@@ -95,6 +96,12 @@ private Object get(String dataId) throws IllegalAccessException {
         Class propertyClass = getPropertyClass(getPropertyPrefix(dataId));
         if (null != propertyClass) {
             Object propertyObject = SpringUtils.getBean(propertyClass);
+
+            if (propertyObject instanceof ServiceProperties &&
+                    dataId.equals(StarterConstants.SERVICE_PREFIX + "." + NORMALIZED_KEY_GROUPLIST)) {
+                return getGroupListForMap((ServiceProperties) propertyObject);
+            }
```

Comment:
Is it compliatable with the following config?
seata.service.vgroup-mapping=default 
 seata.service.grouplist=127.0.0.1:8091
Can you unify the style as map?

Reply:
Old-style transaction grouping configuration is no longer needed.